docs/guides/how_to_guides/creating_batches/how_to_create_a_batch_request_using_an_active_data_connector.rst

```diff
@@ -0,0 +1,269 @@
+
+.. warning::
+
+  ``DataConnector`` is a concept that was introduced in the V3 API, so the following configuration is not supported by the V2 (batch_kwargs) API.
+
+This guide demonstrates how to get a :ref:`batch <reference__core_concepts__glossary__datasources>` of data that Great Expectations can validate from a filesystem using an active ``DataConnector``.  For this how-to-guide, we will be using a ``ConfiguredAssetFilesystemDataConnector``.
+You can read more about the differences between ``ConfiguredAssetDataConnector`` and ``InferredAssetDataConnectors`` :ref:`here <which_data_connector_to_use>`.
+
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+    - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+    - :ref:`Understand the basics of Datasources in 0.13 or later <reference__core_concepts__datasources>`
+    - Learned how to configure a :ref:`DataContext using test_yaml_config <how_to_guides_how_to_configure_datacontext_components_using_test_yaml_config>`
+
+Steps
+-----
+
+If you have the following ``reports`` directory in your filesystem, and you want to treat ``*.csv`` files as batches within the ``reports`` DataAsset:
+
+  .. code-block:: bash
+
+    reports/a-1.csv
+    reports/b-2.csv
+    reports/c-3.csv
+    reports/d-4.csv
+    reports/e-5.csv
+
+
+1. **Load or create a DataContext**
+
+  .. code-block:: python
+
+    import great_expectations as ge
+    from great_expectations.cli.datasource import sanitize_yaml_and_save_datasource
+    from great_expectations.core.batch import BatchRequest
+
+    context = ge.get_context()
+
+
+2. **Configure a Datasource**
+
+  In the following configuration, a Datasource is configured with a ``PandasExecutionEngine`` and ``ConfiguredAssetFilesystemDataConnector``. The DataConnector is configured with a single DataAsset named ``my_reports``. It has the ``base_directory`` set to ``reports/`` and the regex ``pattern`` is set to capture two ``group_names``, ``name`` and ``num``.
+
+  .. code-block:: python
+
+    config = f"""
+      name: mydatasource
+      class_name: Datasource
+      execution_engine:
+        class_name: PandasExecutionEngine
+      data_connectors:
+        my_data_connector:
+          module_name: great_expectations.datasource.data_connector
+          class_name: ConfiguredAssetFilesystemDataConnector
+          base_directory: /
+          glob_directive: "*.csv"
+          assets:
+            my_reports:
+              base_directory: reports/
+              pattern: (.+)-(\\d.*)\\.csv
+              group_names:
+                - name
+                - num
+
+    """
+
+
+3. **(Optional) run** ``test_yaml_config()`` **to ensure that your configuration is working.**
+
+  .. code-block:: python
+
+    context.test_yaml_config(
+        yaml_config=config
+    )
+
+
+  If the configuration is correct you should see output similar to this.  Also, your DataConnector will now be active.
```

Comment:
Could we add a note about what it means for the DataConnector to now be active? How does this compare with sanitizing/saving the config and re-instantiating the DataContext?